lib/ansible/module_utils/network/eos/eos.py

```diff
@@ -232,7 +232,7 @@ def load_config(self, commands, commit=False, replace=False):
 
         if not all((bool(use_session), self.supports_sessions)):
             if commit:
-                return self.configure(self, commands)
+                return self.configure(commands)
```

Comment:
There is another place where this function is called. Would you mind fixing that also? https://github.com/eakman/ansible/blob/2a22085b2b4af6aa9de4b82311e9586d75611bb9/lib/ansible/module_utils/network/eos/eos.py#L423
Thanks.

Reply:
Ok, I made the requested changes.